web/gui/dashboard-react.js

```diff
@@ -0,0 +1,250 @@
+  min_width: null,                // the chart minimum width - can be null
+  library: 'dygraph',             // the graphing library to use
+  method: 'average',              // the grouping method
+  before: 0,                      // panning
+  after: -600,                    // panning
+  pixels_per_point: 1,            // the detail of the chart
+  fill_luminance: 0.8             // luminance of colors in solid areas
+};
+
+
+
+// ----------------------------------------------------------------------------------------------------------------
+// global options
+
+NETDATA.options = {
+  pauseCallback: null,            // a callback when we are really paused
+
+  pause: false,                   // when enabled we don't auto-refresh the charts
+
+  targets: [],                    // an array of all the state objects that are
+                                  // currently active (independently of their
+                                  // viewport visibility)
+
+  updated_dom: true,              // when true, the DOM has been updated with
+                                  // new elements we have to check.
+
+  auto_refresher_fast_weight: 0,  // this is the current time in ms, spent
+                                  // rendering charts continuously.
+                                  // used with .current.fast_render_timeframe
+
+  page_is_visible: true,          // when true, this page is visible
+
+  auto_refresher_stop_until: 0,   // timestamp in ms - used internally, to stop the
+                                  // auto-refresher for some time (when a chart is
+                                  // performing pan or zoom, we need to stop refreshing
+                                  // all other charts, to have the maximum speed for
+                                  // rendering the chart that is panned or zoomed).
+                                  // Used with .current.global_pan_sync_time
+
+  on_scroll_refresher_stop_until: 0, // timestamp in ms - used to stop evaluating
+  // charts for some time, after a page scroll
+
+  last_page_resize: Date.now(),   // the timestamp of the last resize request
+
+  last_page_scroll: 0,            // the timestamp the last time the page was scrolled
+
+  browser_timezone: 'unknown',    // timezone detected by javascript
+  server_timezone: 'unknown',     // timezone reported by the server
+
+  force_data_points: 0,           // force the number of points to be returned for charts
+  fake_chart_rendering: false,    // when set to true, the dashboard will download data but will not render the charts
+
+  passive_events: null,           // true if the browser supports passive events
+
+  // the current profile
+  // we may have many...
+  current: {
+    units: 'auto',              // can be 'auto' or 'original'
+    temperature: 'celsius',     // can be 'celsius' or 'fahrenheit'
+    seconds_as_time: true,      // show seconds as DDd:HH:MM:SS ?
```

Comment:
I do not know whether this variable name is reflecting correctly what it is proposed, the time is by I.S. defined as seconds, so "seconds_as_time" it is something hoped.

Reply:
@thiagoftsm i agree it's confusing, but i think i need to explain few things :) This file (dashboard-react) is a copy-paste of some constants from the old dashboard. There are a lot of things that can be done with this, but I cannot do everything at once. So my idea is to first migrate the logic, chart creation, state handling to the new app. I'm refactoring about 60-80% of the app code, so when i have a choice to just do copy/paste (with minimal tweaks), then i'm doing it so the refractor will finish sooner. What do you think of that approach? Of course i can keep a note of your comment, this is definitely something we want to fix

Reply:
dunno,`seconds_as_time` sounds not great, not terrible for me :smile: 	
meaning is `show seconds as time or not`

Reply:
@thiagoftsm check the dashboard `Settings->Localte->Convert seconds to time?`

Reply:
Your last comment makes more sense for me, I think it was my physicist side, instead my developer time that wrote that comment. :smile: